### ISSUE TITLE
New releases?

### ISSUE THREAD

**Issue by ad-m** [ckknight/async-json#15]
Version available in repository are not in sync with version available on npm. It contains pretty import features eg. Promise support. 

@ckknight what do you think about new release?

**Comment by fijiwebdesign** [comment]
I agree. This is a very useful project and the promises support is a huge feature. Would rather use the npm build.
